Blockzilla/Modules/WebView/WebViewController.swift

```diff
@@ -253,6 +253,12 @@ class WebViewController: UIViewController, WebController {
         browserView.evaluateJavaScript(javascript, completionHandler: completion)
     }
 
+    func evaluatePDFContentType(_ onCompletion: @escaping (Bool) -> Void) {
+        evaluate("document.contentType") { result, _ in
+            onCompletion(result is String && (result as? String) == "application/pdf")
```

Comment:
Completion with Booleans is like having 2 separate methods. Something like `completion(documentType)` is more descriptive.

Reply:
OK, I will add this change. Thank you!